src/actions/admin.ts

```diff
@@ -319,6 +319,30 @@ export function uploadPrivateSamlCertificate(fileData: File): ActionFunc {
     });
 }
 
+export function uploadPublicLdapCertificate(fileData: File): ActionFunc {
+    return bindClientFunc({
+        clientFunc: Client4.uploadPublicLdapCertificate,
+        onRequest: AdminTypes.UPLOAD_LDAP_PUBLIC_REQUEST,
```

Comment:
Unless you're specifically waiting for the _REQUEST and _FAILURE actions, we've stopped using them since they add a lot of boilerplate and cause extra redux changes when we rarely ever looked at their state outside of tests.

In the above tests, you can instead get a result object by `awaiting` on the action.